### PR TITLE
Fix spelling of "Prefer"

### PR DESCRIPTION
**Pull request by davkean**

### CODE REVIEW

**Comment by sharwell** [comment]
@tmeschter @nguerrera This is one of the cases that led to my frustration with a similar localization approach in the past.

**Comment by davkean** [comment]
@tmeschter and @nguerrera I ran what was suggested in the error message and I get thousands of errors:

```
"E:\roslyn\Roslyn.sln" (UpdateXlf target) (1) ->
"E:\roslyn\src\Tools\RoslynPublish\RoslynPublish.csproj" (UpdateXlf target) (151) ->
  E:\roslyn\src\Tools\RoslynPublish\RoslynPublish.csproj : error MSB4057: The target "UpdateXlf" does not exist in the project.
```

I'll guess I'll use the environment variable?

**Comment by nguerrera** [comment]
@davkean You shouldn't need the variable, it should be set for you on local build. Did you build? 

@tmeschter I hadn't considered the difficulty in running /t:UpdateXlf across a solution. An empty "UpdateXlf" target in props could unblock that.

**Comment by tmeschter** [comment]
@davkean Just running "build.cmd" by itself will fix things up. `/t:UpdateXlf` is set automatically for those projects needing localization.

@nguerrera What's the difficulty in running `/t:UpdateXlf` across a solution?

**Comment by tmeschter** [comment]
@davkean Is it possible the baseline for your change predates the Loc change?

File: src/VisualStudio/Core/Def/ServicesVSResources.resx [hunk 1]
```diff
@@ -960,6 +960,6 @@ Additional information: {1}</value>
     <value>Changes are not allowed while code is running.</value>
   </data>
   <data name="Prefer_local_function_over_anonymous_function" xml:space="preserve">
-    <value>Prefere local function over anonymous function</value>
+    <value>Prefer local function over anonymous function</value>
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx]: Ugh.  i suck.

**Comment by nguerrera** [comment]
@tmeschter I imagine projects in the solution that don't reference XliffTasks will trigger the missing error target.

**Comment by davkean** [comment]
> Is it possible the baseline for your change predates the Loc change?
Bingo. To save data (because I'm on 4G) I didn't fetch. Doing that now.

**Comment by davkean** [comment]
This rabbit hole now wants me to download and install 2.2 .NET Core SDK - I'll give up on this change until I have real internet back.

**Comment by Neme12** [comment]
this was fixed in #24808

**Comment by davkean** [comment]
Thanks!

**Pull request closed without merging**